src/routes/Wallets/routes/Backup/WalletsBackupView.js

```diff
@@ -36,6 +49,19 @@ class WalletsBackupView extends Component<Props> {
     openView(params.walletId)
   }
 
+  componentWillReceiveProps(nextProps: Props) {
+    const {
+      items,
+      params,
+    }: Props = nextProps
+
+    const foundWallet: ?Wallet = items.find((w: Wallet): boolean => (w.id === params.walletId))
+
+    if (foundWallet && (foundWallet.type !== this.state.type)) {
```

Comment:
always is a good style to name such expressions e.g.
`const isWalletStateTyped =( findWallet.type !== this.state.type)` <- much better to understand what is going on